recipe/meta.yaml

```diff
@@ -25,25 +24,32 @@ requirements:
   run:
     - python
     - numpy >=1.17.3
-    - scipy >=1.3.2
+    - scipy >=1.5.0
     - scikit-learn >=1.0.2
     - joblib >=1.1.1
-  run_constrained:
     - threadpoolctl >=2.0.0
```

Comment:
threadpoolctl isn't really required. It can be used, but it's not required. Users have to import `threadpoolctl` in their code to actually make use of it. Which is why it was in `run_constrained` and should stay like this.

For example:
```
>>> from threadpoolctl import threadpool_limits
>>> import numpy as np

>>> with threadpool_limits(limits=1, user_api='blas'):
...     # In this block, calls to blas implementation (like openblas or MKL)
...     # will be limited to use only one thread. They can thus be used jointly
...     # with thread-parallelism.
...     a = np.random.randn(1000, 1000)
...     a_squared = a @ a
```

Reply:
yes, you're right. Strangely, the upstream developer [lists it ](https://github.com/scikit-learn-contrib/imbalanced-learn/blob/0a659af54c3b97969ea23ff22a8eebff3696d069/imblearn/_min_dependencies.py#L22) as an install dependency.